.github/workflows/build-and-test-tool.yml

```diff
@@ -14,7 +14,7 @@ on:
       - future
 
 env:
-  dotnet_sdk_version: '7.x'
+  dotnet_sdk_version: '8.0.100-preview.5.23303.2'
```

Comment:
Just remember that the repos referencing this workflow have to be ready for the dotnet-version